lib/src/migrators/module.dart

```diff
@@ -522,45 +522,46 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
-      _patchNamespaceForFunction(node, declaration, (namespace) {
-        addPatch(patchBefore(node.name, '$namespace.'));
-      });
-    }
+    if (node.namespace == null) {
+      if (references.sources.containsKey(node)) {
+        var declaration = references.functions[node];
+        _unreferencable.check(declaration, node);
+        _renameReference(nameSpan(node), declaration);
+        _patchNamespaceForFunction(node, declaration, (namespace) {
+          addPatch(patchBefore(node.name, '$namespace.'));
+        });
+      }
 
-    if (node.name.asPlain == "get-function") {
-      var declaration = references.getFunctionReferences[node];
-      _unreferencable.check(declaration, node);
-      _renameReference(getStaticNameForGetFunctionCall(node), declaration);
+      if (node.name.asPlain == "get-function") {
+        var declaration = references.getFunctionReferences[node];
+        _unreferencable.check(declaration, node);
+        _renameReference(getStaticNameForGetFunctionCall(node), declaration);
 
-      // Ignore get-function calls that already have a module argument.
-      var moduleArg = node.arguments.named['module'];
-      if (moduleArg == null && node.arguments.positional.length > 2) {
-        moduleArg = node.arguments.positional[2];
-      }
-      if (moduleArg != null) return;
+        // Ignore get-function calls that already have a module argument.
+        var moduleArg = node.arguments.named['module'];
+        if (moduleArg == null && node.arguments.positional.length > 2) {
+          moduleArg = node.arguments.positional[2];
+        }
+        if (moduleArg != null) return;
+
+        // Warn for get-function calls without a static name.
+        var nameArg =
+            node.arguments.named['name'] ?? node.arguments.positional.first;
+        if (nameArg is! StringExpression ||
+            (nameArg as StringExpression).text.asPlain == null) {
+          emitWarning(
+              "get-function call may require \$module parameter", nameArg.span);
+          return;
+        }
 
-      // Warn for get-function calls without a static name.
-      var nameArg =
-          node.arguments.named['name'] ?? node.arguments.positional.first;
-      if (nameArg is! StringExpression ||
-          (nameArg as StringExpression).text.asPlain == null) {
-        emitWarning(
-            "get-function call may require \$module parameter", nameArg.span);
-        return;
+        _patchNamespaceForFunction(node, declaration, (namespace) {
+          var beforeParen = node.span.end.offset - 1;
+          addPatch(Patch(node.span.file.span(beforeParen, beforeParen),
+              ', \$module: "$namespace"'));
+        }, getFunctionCall: true);
       }
-
-      _patchNamespaceForFunction(node, declaration, (namespace) {
-        var beforeParen = node.span.end.offset - 1;
-        addPatch(Patch(node.span.file.span(beforeParen, beforeParen),
-            ', \$module: "$namespace"'));
-      }, getFunctionCall: true);
```

Comment:
It's hard to tell for sure, but nothing actually changed in this block other than the indentation and calling `super.visitFunctionExpression()` below it instead of above it, right?

Reply:
Yes